src/test/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderControllerTest.java

```diff
@@ -1,64 +1,106 @@
 package uk.gov.justice.probation.courtcaseservice.controller;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.justice.probation.courtcaseservice.application.FeatureFlags;
+import uk.gov.justice.probation.courtcaseservice.controller.model.ConvictionResponse;
 import uk.gov.justice.probation.courtcaseservice.controller.model.RequirementsResponse;
+import uk.gov.justice.probation.courtcaseservice.service.BreachService;
+import uk.gov.justice.probation.courtcaseservice.service.ConvictionService;
 import uk.gov.justice.probation.courtcaseservice.service.OffenderService;
 import uk.gov.justice.probation.courtcaseservice.service.model.ProbationRecord;
 import uk.gov.justice.probation.courtcaseservice.service.model.Requirement;
+import uk.gov.justice.probation.courtcaseservice.service.model.UnpaidWork;
 
 import java.util.Collections;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 
 @ExtendWith(MockitoExtension.class)
 public class OffenderControllerTest {
     public static final String CRN = "CRN";
     public static final String CONVICTION_ID = "CONVICTION_ID";
+    static final Long SOME_EVENT_ID = 1234L;
 
     @Mock
-    private OffenderService service;
+    private OffenderService offenderService;
     @Mock
     private ProbationRecord expectedProbationRecord;
     @Mock
     private Requirement expectedRequirement;
+    @Mock
+    private ConvictionService convictionService;
+    @Mock
+    private BreachService breachService;
```

Comment:
Missing test. This field is only used in the constructor for OffenderController. When I run OffenderControllerTest with coverage, there are no tests over getBreach.